src/schemes/competition.ts

```diff
@@ -657,24 +667,15 @@ export class CompetitionSuggestion {
   }
 
   public async getPosition() {
+    console.warn(`This method is deprecated - please use the positionInWinnerList from the proposal state`)
     const suggestionState = await this.state().pipe(first()).toPromise()
-    const proposal = new Proposal(suggestionState.proposal, this.context)
-    const proposalState = await proposal.state().pipe(first()).toPromise()
-    const scheme = new CompetitionScheme(proposalState.scheme.id, this.context)
-    const competitionContract = await scheme.getCompetitionContract()
-    const transaction = competitionContract.methods.getOrderedIndexOfSuggestion(suggestionState.suggestionId)
-    const result = await transaction.call()
-    const index = Number(result)
-    return index
+    return suggestionState.positionInWinnerList
   }
+
   public async isWinner() {
+    console.warn(`This method is deprecated - please use the positionInWinnerList !== from the proposal state`)
```

Comment:
`isWinner` is nice intuitive sugar. How about leave it in and just make it return `suggestionState.positionInWinnerList === null`?  Add it to `ICompetitionSuggestionState`, an interface which doesn't exist, but seems like it should.

Reply:
yes https://github.com/daostack/client/issues/374